modules/nw-udn-support-matrix-primary-secondary.adoc

```diff
@@ -0,0 +1,53 @@
+//module included in the following assembly:
+//
+// *networkking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="support-matrix-for-primary-secondary-networks_{context}"]
+= Support matrix for primary and secondary networks
+
+The following table outlines supported features of primary and secondary networks created by `UserDefinedNetwork` (UDN) and other secondary networks. For more information on, the limitations of user-defined networks see: xref:../networking/ . For more information on secondary networks that are created through network attachement defintions using multus, see xref:../networking/.
+
+[cols="<.^3,^.^1,^.^1,^.^1,^.^1,^.^1,^.^1,^.^1,^.^1,^.^1,^.^1"]
+|====
+|Network type |definition |role |services |east-west traffic |north-sourth traffic |Persistent IPs |`NetworkPolicy`| `MultiNetworkPolicy` |Ingress |Egress IP
+
+|UDN primary
```

Comment:
This is for 4.17 right?  Then service and networkpolicy is not ready in 4.17.
Can you find some DEV to review firstly, thanks!

Reply:
Services and Netpol have been commented out.